plugins/woocommerce/tests/php/src/Internal/Admin/Settings/PaymentsProviders/PaymentGatewayTest.php

```
@@ -2679,4 +2679,177 @@ public function data_provider_invalid_country_codes_for_logging(): array {
+	public function test_is_in_test_mode_survives_non_scalar_option_values( $value ) {
+		// Arrange.
+		$fake_gateway = new class( 'junk_option_gateway', array() ) extends FakePaymentGateway {
+			/**
+			 * The value to return.
+			 *
+			 * @var mixed
+			 */
+			public $value;
+
+			// phpcs:disable Squiz.Commenting.FunctionComment.Missing, Squiz.Commenting.FunctionComment.MissingParamTag
+			public function get_option( $key, $empty_value = null ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)